libs/src/sanityChecks/sanitizeNodes.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Sanitize user.creator_node_endpoint
+ * Goal: Make it so we never end up in a state like creator_node_endpoint = "https://cn1.co,,"
+ */
+const sanitizeNodes = async (libs, secondaries) => {
```

Comment:
This all looks fine codewise, tho I am slightly nervous that we don't understand why this happens or what other weird states with this string manipulation we might get into

Reply:
fair point. let's definitely discuss how we can build the proper pattern here next week